util/config.py

```diff
@@ -1,20 +1,41 @@
 from pathlib import Path
 import toml
 
-def get_functions_csv_path() -> Path:
-    return get_repo_root() / load_toml()["functions_csv"]
 
-def get_base_elf() -> Path:
-    return get_repo_root() / 'data/main.elf'
+def get_repo_root() -> Path:
+    return Path(__file__).resolve().parent.parent.parent.parent
+
+def load_toml() -> toml:
+    return toml.load(get_repo_root() / "tools" / "config.toml")
+
+def get_default_version() -> str:
+    return load_toml().get("default_version")
+
+def get_functions_csv_path(version = get_default_version()) -> Path:
```

Comment:
Can we avoid reloading the entire config more than once? This seems a bit silly.